src/data/fantom/spookyLpPools.json

```diff
@@ -1,4 +1,23 @@
 [
+  {
+    "name": "boo-dai-fhm",
+    "address": "0xd77fc9c4074b56Ecf80009744391942FBFDDd88b",
+    "decimals": "1e18",
+    "poolId": 0,
+    "chainId": 250,
+    "lp0": {
+      "address": "0x8D11eC38a3EB5E956B052f67Da8Bdc9bef8Abf3E",
+      "oracle": "tokens",
+      "oracleId": "DAI",
```

Comment:
For dummy pools, move them here https://github.com/beefyfinance/beefy-api/blob/master/src/data/archive/oldLpPools.json